ecl2df/ecl2csv.py

```diff
@@ -207,6 +207,14 @@ def get_parser() -> argparse.ArgumentParser:
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
         description=("Each row represents an edge in the GRUPTREE at a specific date."),
     )
+    subparsers_dict["wellcompletiondata"] = subparsers.add_parser(
+        "wellcompletiondata",
+        help="Extract well completion data",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description=(
+            "Each row represents a completion, aggregated from layer to zone."
```

Comment:
Testing my understanding of this description: Is this the same as "Each row represents an aggregated completion over a zones"?

Reply:
Yes, I guess, or maybe aggregated completion FOR zones ? For each group of equal well, date and zone, the compdat rows are aggregated to one row.